docs/manual/developer/06_contributing_with_content.md

```diff
@@ -610,15 +610,22 @@ Tips:
 
 ### Checks
 
-Checks are used to evaluate a Rule. They are written using a custom OVAL
-syntax and are stored as xml files inside the *checks/oval* directory
-for the desired platform. During the building process, the system will
-transform the checks in OVAL compliant checks.
+Checks are used to evaluate a Rule.
+They are written using a custom OVAL syntax and during the building process
+the system will transform them into OVAL compliant checks.
```

Comment:
```suggestion
They are written using a custom OVAL syntax and transformed by the system
during the building process into OVAL compliant checks.
```

Reply:
Thanks, I've edited your suggestion a bit applied it.